File: vlantana-wms-backend-laravel/app/Models/Company.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\HasMany;

class Company extends Model
{
    use HasFactory;

    protected $fillable = [
        'company_name'
    ];

    public function products() : HasMany {
        return $this->hasMany(Product::class);
    }

    public function pallets() : HasMany {
        return $this->hasMany(Pallet::class);
    }

    public function orders() : HasMany {
        return $this->hasMany(Order::class);
    }

    public function users() : HasMany {
        return $this->hasMany(User::class);
    }
}

File: vlantana-wms-backend/WMS.Web/Models/Business/Company.cs
using System.ComponentModel.DataAnnotations;
using vlantana_wms_backend.Models.Auth;

namespace vlantana_wms_backend.Models.Business
{
    public class Company
    {
        [Key]
        public int CompanyId { get; set; }

        [Required]
        [StringLength(50)]
        public string? CompanyName { get; set; }
        public virtual ICollection<Product>? Products { get; set; } // Has a lot of products
        public virtual ICollection<Pallete>? Palletes { get; set; } // Has a lot of pallets that have the products
        public virtual ICollection<Order>? Orders { get; set; } // Has a lot of orders
        public virtual ICollection<User>? Users { get; set; } // Has a lot of employees/users
    }
}
